### PR TITLE
MOB-1521 No internet screen refresh

### PR DESCRIPTION
**Pull request by d4r1091**
[MOB-1521](https://ecosia.atlassian.net/browse/MOB-1521)

This ticket has been instrumental in understanding a bit more about how the codebase collaborates with internal definitions of HTML/CSS.

This PR updates the copies and fixes the image on the _"No Internet screen"_.
This page is indeed a static web page.
As part of the copy update work, I've also improved the methodology for loading the error image within the web page.
We were grabbing a `png` without the standard 1x/2x/3x resolutions that Apple provided.
So, once I fixed the retrieval, I realized that the image was blurred.
Fix applied by importing the `pdf` of the same image as part of our asset and loading the data via `UIImage.pngData()`

Note: in Figma, the padding looks different. I didn't want to start modifying the `HTML/CSS` made by Firefox as I didn't think it would add much value to the work.

Next step would be to understand in practice how the localization process works.

The result is in the screenshot attached.

IMG_D6C86E2A5EF1-1

### CODE REVIEW

File: Client/Ecosia/Extensions/ErrorPageHandler+Ecosia.swift [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+/// These strings modify the default title, message and button text
```
Comment (ecotopian) [inline Client/Ecosia/Extensions/ErrorPageHandler+Ecosia.swift:7]: 📖 great habit to add explanatory documentation for new classes 👍

File: Client/Frontend/InternalSchemeHandler/InternalSchemeHandler.swift [hunk 2]
```diff
@@ -44,9 +44,8 @@ class InternalSchemeHandler: NSObject, WKURLSchemeHandler {
                 urlSchemeTask.didReceive(data)
                 urlSchemeTask.didFinish()
                 return true
-            } else if path.hasSuffix("EcosiaError.png"),
-                      let res = Bundle.main.url(forResource: path, withExtension: nil),
-                      let data = try? Data(contentsOf: res) {
+            } else if path.hasSuffix("EcosiaErrorPlaceholderPath.png"),
+                      let data = UIImage(named: "noInternet")?.pngData() {
```
Comment (ecotopian) [inline Client/Frontend/InternalSchemeHandler/InternalSchemeHandler.swift:48]: great fix 💪

File: Client/Ecosia/L10N/String.swift [hunk 3]
```diff
@@ -56,6 +56,9 @@ extension String {
         case networkError = "Network error!"
         case new = "New"
         case noConnectionMessage = "We couldn't verify your link. Please check your internet connection and try again."
+        case noConnectionNSURLErrorTitle = "No connection"
```
Comment (ecotopian) [inline Client/Ecosia/L10N/String.swift:59]: I see that you introduced those three string. I wonder if they are in some form existing in FF's l10n already. If so we don't need to provide our own translation and use the given ones, which in this case are missing. But I am happy to explain to you how to get those 🚀

**Review by ecotopian — CHANGES_REQUESTED** [review]
Great work!! This was a tricky quite tangled task, and without any pointer from my side you find the right spots in the codebase. That is impressive! 🚀 👏 

Still before I can approve you need to either provide our own l10n for the added strings or find them in MZ's code to continue. You find docs about how to use TF here:
https://github.com/ecosia/ios-browser#translations

I am happy to assist you there.

**Comment by d4r1091** [comment]
Checked the translations this morning @ecotopian. All the required languages have been translated.
You can check in the last commit 👍

**Pull request merged**